docs/_static/config_examples/example-vs-resource-source-address-translation.configmap.yaml

```diff
@@ -0,0 +1,35 @@
+kind: ConfigMap
```

Comment:
Just a nitpick, but I would shorten the name of this file since it's really long. Something like 'source-addr-translation.configmap.yaml'.

Reply:
sounds good will do.